### PR TITLE
Set variables for robust running in RStudio

### PR DESCRIPTION
**Pull request by krlmlr**
Borrowed from `verify_output()`.

### CODE REVIEW

File: R/check.R [hunk 1]
```diff
@@ -89,6 +89,8 @@ revdep_env_vars <- function(force_suggests = FALSE) {
     "_R_CHECK_FORCE_SUGGESTS_" = as.character(force_suggests),
     "RGL_USE_NULL" = "TRUE",
     DISPLAY = "",
+    RSTUDIO = 0,
```
Comment (gaborcsardi) [inline R/check.R]: Why would we set this to zero?

Comment (gaborcsardi) [reply R/check.R]: Or this is for the subprocesses?

Comment (krlmlr) [reply R/check.R]: This is for subprocesses which might pick up this settings. I'm getting unstable results from RStudio, running with this PR now. We'll see if it helps.

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**